### PR TITLE
Changed default initialDate

### PR DESCRIPTION
**Pull request by Aeonax**

### CODE REVIEW

File: src/DatePickerWithData.js [hunk 1]
```diff
@@ -37,6 +37,10 @@ function toWeeks(availability) {
   )(availability);
 }
 
+function calendarMonthDate(initialDate, selectedDate): Date {
```
Comment (Aeonax) [inline src/DatePickerWithData.js]: may be should rename it...

Comment (fmnoise) [reply src/DatePickerWithData.js]: I'm not sure it's worth extracting to function
what do you think @bskog ?

Comment (bskog) [reply src/DatePickerWithData.js]: Agree. And I think you can make it even shorter, like this: `calendarMonthDate : new Date(initialDate || selectedDate || new Date())`

**Comment by fmnoise** [comment]
Looks good to me, @bskog ?

**Pull request merged**